.circleci/config.yml

```diff
@@ -557,6 +553,23 @@ jobs:
       ENVOY_VERSIONS: "1.11.1"
     steps: *ENVOY_INTEGRATION_TEST_STEPS
 
+  # run tests on vault ca provider integration tests
+  vault-ca-provider:
```

Comment:
Possible future improvement: pick a series of vault versions and duplicate (or templateize) this for varying values of `VAULT_BINARY_VERSION` so that we can test the last N major releases against consul.

Reply:
Yup! I will make sure we document this before closing out #6535